src/service/viewer-impl.js

```diff
@@ -221,6 +221,10 @@ export class Viewer {
     this.performanceTracking_ = this.params_['csi'] === '1';
     dev.fine(TAG_, '- performanceTracking:', this.performanceTracking_);
 
+    this.shareTrackingIncomingFragment_ = this.params_['share-tracking'];
```

Comment:
But `#getParam` is [exactly](https://github.com/ampproject/amphtml/blob/master/src/service/viewer-impl.js#L406-L408) this code...